service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
```

Comment:
Right, yes. I'm fine with running the tests as one script (or one command). But I think it's important to have test setup and test running as two separate stages. For example, if you want to run test setup on your personal dev machine, you should be able to do it without running tests. Travis even has the two stages of setup and testing separated (you've got the `install` field for setup, and the `script` field for testing). I'm not sure I see a reason to stick both pieces of functionality into one script.